src/js/stores/app-store.js

```diff
@@ -40,14 +40,16 @@ var _groups = [];
 var _releasesRepo = [];
 var _uploadInProgress = false;
 const _hostAddress = mender_environment && mender_environment.hostAddress ? mender_environment.hostAddress : null;
+const _IntegrationVersion = mender_environment && mender_environment.integrationVersion ? mender_environment.integrationVersion : 'master';
 const _MenderVersion = mender_environment && mender_environment.menderVersion ? mender_environment.menderVersion : 'master';
 const _menderArtifactVersion = mender_environment && mender_environment.menderArtifactVersion ? mender_environment.menderArtifactVersion : 'master';
 const _menderDebPackageVersion = mender_environment && mender_environment.menderDebPackageVersion ? mender_environment.menderDebPackageVersion : 'master';
 var _demoArtifactPort = mender_environment && mender_environment.demoArtifactPort ? mender_environment.demoArtifactPort : 85;
 var _globalSettings = {};
 
 const _versionInformation = {
-  Mender: mender_environment.menderVersion,
+  Integration: mender_environment.integrationVersion,
+  MenderClient: mender_environment.menderVersion,
```

Comment:
the key of the object is directly shown in the GUI (or omitted if no version is set) - to align it with the other keys I would prefer this as 'Mender-Client'

Reply:
Fixed!